services/user-tenant-service/pom.xml

```diff
@@ -117,6 +121,21 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.flywaydb</groupId>
+                <artifactId>flyway-maven-plugin</artifactId>
+                <!--<version>9.5.1</version> --><!-- Replace with the Flyway version you are using -->
+                <configuration>
+                    <url>jdbc:postgresql://postgres:5432/userTenant</url>
+                    <user></user>
+                    <password></password>
+                    <schemas>
+                        <schema>public</schema>
+                    </schemas>
+<!--             specify custom location if required, default location is db/migration
+<locations>services/user-tenant-service/src/main/resources/db/migration</locations>-->
+                </configuration>
```

Comment:
you don't need to dd config here, there should be a way to pass it from mvn command 

i am not sure this will work or not but you can try it this way 

`mvn flyway:migrate -Dflyway.url=jdbc:postgresql://localhost:3306/userTenant -Dflyway.user=myuser -Dflyway.password=mypassword Dflyway.schemas=public`